src/runtime/runtime.js

```diff
@@ -27,12 +27,10 @@
   var $TypeError = TypeError;
   var $create = $Object.create;
   var $defineProperty = $Object.defineProperty;
-  var $freeze = $Object.freeze;
+  var {freeze} = $Object;
```

Comment:
It's not clear to me why this one is `freeze` while all other functions of `$Object` are eg `$freeze`.